rs/ethereum/ledger-suite-orchestrator/BUILD.bazel

```diff
@@ -161,6 +161,9 @@ rust_ic_test(
         "LEDGER_ARCHIVE_NODE_CANISTER_WASM_PATH": "$(rootpath //rs/ledger_suite/icrc1/archive:archive_canister_u256.wasm.gz)",
     },
     proc_macro_deps = [],
+    tags = [
+        "slow_test",  # since it takes longer than 5 minutes.
+    ],
```

Comment:
@basvandijk : I'm not sure I understand why those tests should be `slow` since according to the chart
[Bazel Test All / Average Event Duration / From Last Week Onwards](https://superset.idx.dfinity.network/explore/?form_data_key=voaCAzP6bu_XAKchDTKMxJAC88H2waxKNOTtAQOX6x-oBUwvee0esRDDLYnS8MRB&dashboard_page_id=Ku6Er4iQBH&slice_id=47) the test `Testing //rs/ethereum/ledger-suite-orchestrator:integration_tests` takes 4m 31.8s, meaning less than 5 minutes. Am I looking at the wrong chart?

Reply:
Hi @gregorydemay, I'm using the chart [Bazel Test All / P90 Test Duration / From Last Week Onwards](https://superset.idx.dfinity.network/explore/?slice_id=48). Unfortunately there's an issue that prevents unauthenticated users from accessing its dataset. I hope to fix that this week. But here's a screenshot:
![Screenshot 2024-10-16 at 08 02 15](https://github.com/user-attachments/assets/b39fde93-cbd6-4d84-bfbf-cf6565c7f38e)

There is one big difference between the two chart: the one I'm using uses the 90th percentile of test duration while the one you looked at uses the average duration. So what this likely means is that the distribution of durations for `//rs/ethereum/ledger-suite-orchestrator:integration_tests` skews to the left brining the average down while having a long tail to the right brining the 90th percentile up.

Reply:
Thanks a lot @basvandijk for the analysis and the pointers! I'm a bit puzzled by the time those tests take since those are state machine tests, so ideally they should be fast and I would like them to run on PRs if something changed. Would it be an option that I create a follow-up PR, where I basically split those tests in 2 targets to have a better idea what's going there and remove the `slow_test` label for those 2 targets (assuming that each target runs under 5 minutes)?

Reply:
Sounds good!